client/services/FeedbackGeneratorService.js

```diff
@@ -183,13 +184,17 @@ tie.factory('FeedbackGeneratorService', [
 
           var correctnessTests = task.getCorrectnessTests();
           var observedOutputs = codeEvalResult.getCorrectnessTestResults();
+          var stdOutputs = codeEvalResult.getOutput();
           for (i = 0; i < correctnessTests.length; i++) {
             var observedOutput = observedOutputs[i];
+            var outputCount = stdOutputs.length / correctnessTests.length;
```

Comment:
It's not always going to be the case that the output for each test case is of equal length. You might need to find a way to catch each of them somewhere.